src/test/java/io/logz/MockLogzioBulkListener.java

```diff
@@ -149,10 +153,63 @@ public boolean checkForLogExistence(String token, String type, String loggerName
                 if (!jsonObject.get("message").getAsString().equals(message)) {
                     found = false;
                 }
+
+                if (checkHostname) {
```

Comment:
Isnt that cleaner this way?
I can just assent on that instead of implementing similar logic in every test